dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageContainerForm.java

```diff
@@ -76,6 +79,33 @@ public PageContainerForm deserialize(final JsonParser jsonParser,
                 }
 
                 contentletsNode.forEach((JsonNode contentId) -> containerEntry.addContentId(contentId.textValue()));
+
+                // Parse styleProperties for each contentlet (optional field)
+                final JsonNode stylePropertiesNode = jsonElement.get(STYLE_PROPERTIES_ATTRIBUTE_NAME);
+                if (stylePropertiesNode != null && stylePropertiesNode.isObject()) {
+                    stylePropertiesNode.fields().forEachRemaining(entry -> {
+                        final String contentletId = entry.getKey();
+                        final JsonNode styleProps = entry.getValue();
+                        if (styleProps != null && styleProps.isObject()) {
+                            final Map<String, Object> propsMap = new HashMap<>();
+                            styleProps.fields().forEachRemaining(prop -> {
+                                final JsonNode propValue = prop.getValue();
+                                // Handle different JSON value types
```

Comment:
This looks a bit limited in the number of cases we're handling.
I think we're gonna need a test here